bigbluebutton-html5/imports/ui/components/settings/submenus/notification/component.jsx

```diff
@@ -64,6 +69,7 @@ class NotificationMenu extends BaseMenu {
     } = this.props;
 
     const { settings } = this.state;
+    const isLiveKit = audioManager.bridge?.bridgeName === 'livekit';
```

Comment:
See suggestion above.
```suggestion
    const isLiveKit = AudioManager.bridge?.bridgeName === 'livekit';
```